scripts/init-repo/prerequisities.ps1

```diff
@@ -0,0 +1,88 @@
+####################################################
+# prerequisities - check                           #
+####################################################
+
+function Invoke-InitRepoSectionPrerequisities
+{
+    [CmdLetBinding()]
+    param
+    (
+        [ref]
+        [Parameter(Mandatory=$true)]
+        $success
+    )
+    Write-Output 'Prerequisities check...';
+    # check for WMF 5.0 +
+    if ((Get-Command -Name Get-PackageProvider -ErrorAction SilentlyContinue) -eq $null)
+    {
+        Write-Warning "Unable to find Get-PackageProvider... do you have WMF 5.0 installed?"
+        return;
+    }
+    Write-Output '... WMF 5.0 found.';
+
+    #check for GIT
+    if ((Get-Command -Name git -ErrorAction SilentlyContinue) -eq $null)
+    {
+        $message  = "Unable to find GIT";
+        $LASTEXITCODE = 0;
+        $question = 'We need to install GIT (via Chocolatey). Is this OK?';
+        if (!(PromptUserYN -Message $message -Question $question))
+        {
+            return;
+        }
+        Install-Choco;
+        choco install git.install --force --force-dependencies -y;
+        if ((Get-Command -Name git -ErrorAction SilentlyContinue) -eq $null)
+        {
+            Write-Warning 'Unable to find [git] command. Try restarting the powershell console and run again the whole script.'
+            return;
+        }
+    }
+
+    #check for TF
+    $tfExe = "C:\Program Files (x86)\Microsoft Visual Studio\2019\TeamExplorer\Common7\IDE\CommonExtensions\Microsoft\TeamFoundation\Team Explorer\tf.exe"
```

Comment:
This is bit dangerous or? :finnadie: what if the visual studio is installed in different path or different version?

Reply:
I wanted to use originaly (get-command -name TF) but TF location is not appended to PATH automatically..so I would need to locate/specify the TF path anyway.

I do install TF via 
```
choco install visualstudio2019teamexplorer --force --force-dependencies --package-parameters
```
so this is the path it will be installed to. However I have just realized a different thing. Since I have separated the scripts to multiple files, this is something, that needs to be passed between and we should not rely on global variable...